api/tests/utils/apply-snapshot.test.ts

```diff
@@ -0,0 +1,270 @@
+import knex, { Knex } from 'knex';
+import { getTracker, MockClient, Tracker } from 'knex-mock-client';
+import { snapshotApplyTestSchema } from '../__test-utils__/schemas';
+
+import { applySnapshot } from '../../src/utils/apply-snapshot';
+import { testSnapshotToApply, testSnapshotBefore, testSnapshotToApplyNotNested } from '../__test-utils__/snapshots';
+import { CollectionsService, FieldsService } from '../../src/services';
+import * as getSchema from '../../src/utils/get-schema';
+
+jest.mock('../../src/database/index', () => {
+	return {
+		getDatabaseClient: jest.fn().mockReturnValue('postgres'),
+	};
+});
+jest.requireMock('../../src/database/index');
+
+class Client_PG extends MockClient {}
```

Comment:
This is just to satisfy `SchemaInspectorConstructor` which is called in some of the service constructions